samples/holiday-package-sample/services/samples/holidayPackage/HolidayPackageDetails.bal

```diff
@@ -0,0 +1,189 @@
+                                 "DepartureDate":departureDate,
+                                 "ReturnDate":returnDate,
+                                 "From":from,
+                                 "To":to,
+                                 "price":275
+                             };
+        response.setJsonPayload(flightDetails);
+        _ = response.send();
+    }
+
+    @http:resourceConfig {
+        methods:["POST"],
+        path:"/airline/emirates"
+    }
+    resource flightEmirates (http:Request request, http:Response response) {
+        json receivedPayload = request.getJsonPayload();
+        json departureDate = receivedPayload.departure;
+        json returnDate = receivedPayload.returnDate;
+        json from = receivedPayload.from;
+        json to = receivedPayload.to;
+        json flightDetails = {
+                                 "Flight":"Emirates",
+                                 "DepartureDate":departureDate,
+                                 "ReturnDate":returnDate,
+                                 "From":from,
+                                 "To":to,
+                                 "price":273
+                             };
+        response.setJsonPayload(flightDetails);
+        _ = response.send();
+    }
+
+    @http:resourceConfig {
+        methods:["POST"],
+        path:"/airline/driveSg"
+    }
+    resource driveSg (http:Request request, http:Response response) {
+        json receivedPayload = request.getJsonPayload();
+        json from = receivedPayload.from;
+        json to = receivedPayload.to;
+        json vehicleType = receivedPayload.vehicleType;
+        json vehicleDetails = {
+                                  "Company":"DriveSG",
+                                  "VehicleType":vehicleType,
+                                  "From":from,
+                                  "To":to,
+                                  "price":50
+                              };
+        response.setJsonPayload(vehicleDetails);
+        _ = response.send();
+    }
+
+    @http:resourceConfig {
+        methods:["POST"],
+        path:"/airline/dreamCar"
+    }
+    resource dreamCar (http:Request request, http:Response response) {
+        json receivedPayload = request.getJsonPayload();
+        json from = receivedPayload.from;
+        json to = receivedPayload.to;
+        json vehicleType = receivedPayload.vehicleType;
+        json vehicleDetails = {
+                                  "Company":"DreamCar",
+                                  "VehicleType":vehicleType,
+                                  "From":from,
+                                  "To":to,
+                                  "price":60
+                              };
+        response.setJsonPayload(vehicleDetails);
+        _ = response.send();
+    }
+
+    @http:resourceConfig {
+        methods:["POST"],
+        path:"/airline/sixt"
+    }
+    resource SixT (http:Request request, http:Response response) {
```

Comment:
follow a convention in naming the resources, (others are starting in lower cases)